Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM golang
+FROM php:8.2.0RC4-zts-bullseye AS php-base
+
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
```

Comment:
Maybe should we keep with the choices made in the official image and recommend this script in the documentation but not install it by default?

Just thinking out loud, I've no strong opinion about that.